lib/Remote/DriverCommand.php

```diff
@@ -76,6 +76,7 @@ class DriverCommand
     const GET_ELEMENT_VALUE_OF_CSS_PROPERTY = 'getElementValueOfCssProperty';
     const ELEMENT_EQUALS = 'elementEquals';
     const SCREENSHOT = 'screenshot';
+    const ELEMENT_SCREENSHOT = 'elementScreenshot';
```

Comment:
The W3C spec talls this "Take Element Screenshot" so I'd suggest we go for TAKE_ELEMENT_SCREENSHOT and takeElementScreenshot to match.

Reply:
Agreed, also after you rebase the pull request, please move this constant to the end of the file (we now have there `// W3C specific`).

Reply:
Done